src/utils/getItemLabel.ts

```diff
@@ -0,0 +1,21 @@
+export interface GetItemLabelParams {
+  report: any;
+  labelKey: any;
+  value: any;
+}
+
+export function getItemLabel({ report, labelKey, value }: GetItemLabelParams) {
+  let itemLabelKey = String(labelKey);
+  if (report.meta && report.meta.group_by) {
+    const group_by = report.meta.group_by;
+    for (const key of Object.keys(group_by)) {
+      if (key.indexOf('tag:')) {
+        const tagPrefixKey = 'tag:' + labelKey;
```

Comment:
FYI, we have this tag prefix defined in Query.

https://github.com/project-koku/koku-ui/blob/master/src/api/query.ts#L4

Reply:
Done. Thanks for the pointer 🙇